x-pack/plugins/security_solution/server/lib/risk_engine/schema/risk_score_apis.yml

```diff
@@ -153,7 +203,46 @@ components:
           description: A list of risk scores
           items:
             $ref: '#/components/schemas/RiskScore'
+    RiskEngineStatusResponse:
+      type: object
+      properties:
+        legacy_risk_engine_status:
+          $ref: '#/components/schemas/RiskEngineStatus'
+        risk_engine_status:
+          $ref: '#/components/schemas/RiskEngineStatus'
+    RiskEngineInitResponse:
+      type: object
+      properties:
+        errors: 
+          type: array
+          items:
+            type: string
+        result:
+          type: object
+          properties:
+            riskEngineEnabled:
```

Comment:
Can we snake_case these? The camelCased fields are inconsistent with the other APIs.

Reply:
I can:) just feel a bit wrong that we manipulate camelCase on the server side and clients side, but for API layer should make it different

Reply:
Agreed, but that's always been the way (unless we forget 😅).